boring-sys/build.rs

```diff
@@ -182,10 +182,20 @@ fn main() {
             .cxxflag("-DBORINGSSL_UNSAFE_FUZZER_MODE");
     }
 
-    let bssl_dir = cfg.build_target("bssl").build().display().to_string();
+    #[cfg(feature = "fips")]
+    cfg.define("FIPS", "1");
+
+    let bssl_dir = cfg.build_target("bssl").build();
     let build_path = get_boringssl_platform_output_path();
-    let build_dir = format!("{}/build/{}", bssl_dir, build_path);
-    println!("cargo:rustc-link-search=native={}", build_dir);
+    let build_dir = bssl_dir.join("build").join(build_path);
+    println!(
+        "cargo:rustc-link-search=native={}",
+        build_dir.join("crypto").to_str().unwrap()
+    );
```

Comment:
Seems like this broke linking on Windows (see CI failures)

Reply:
Yeah I was looking into this. I'll need to spin up a windows box to investigate this further but I've at least verified that we are using the correct slashes `/` vs `\` based on the build platform.